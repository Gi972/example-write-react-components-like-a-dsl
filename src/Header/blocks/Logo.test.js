import React from 'react'
import rederer from 'react-test-renderer'

import Logo from './Logo'

describe('<Logo/>', () => {
  it('renders', () => {
    expect(<Logo/>).toMatchSnapshot()
  })
})
