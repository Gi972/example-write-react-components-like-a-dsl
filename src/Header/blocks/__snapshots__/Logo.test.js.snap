// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo/> renders 1`] = `<Logo />`;
